server/modules/category/api/categoryApi.js

```diff
@@ -132,6 +133,28 @@ export const init = app => {
     },
   )
 
```

Comment:
if it's used in rChain, then add this to rChainApi and use /api/rChain/category

Reply:
as you wish